.circleci/config.yml

```diff
@@ -66,6 +66,29 @@ jobs:
             yarn addons-linter coilfirefoxextension@coil.com.xpi
       - *save_cache
 
+  publish-packages:
+    docker:
+      - image: circleci/node:14-buster-browsers
+    steps:
+      - checkout
+      - *restore_cache
+      - *yarn_install
+      - run:
+          name: Configure NPM authentication
+          command: |
+            yarn config set "npmRegistries['https://registry.yarnpkg.com'].npmAuthToken" "$NPM_AUTH_TOKEN" -H
+      - run:
+          name: Build the web-monetization-types package
+          command: |
+            cd packages/web-monetization-types
+            yarn build:ts:verbose
+      - run:
+          name: Publish the web-monetization-types package
+          command: |
+            cd packages/web-monetization-types
+            yarn npm publish --access public
```

Comment:
![image](https://user-images.githubusercontent.com/525211/121477092-9b048c80-c9f1-11eb-8153-113d127bdfcc.png)

I find this "stalled" yellow dot annoying somehow as am so used to waiting for the green tick. 

I wonder if using the following `yarn npm publish` option and removing the hold may be better:

```
  --tolerate-republish    Warn and exit when republishing an already existing version of a package
```

Reply:
Its a really annoying side effect of the hold. You could just auto publish on main by filtering the workflow to only the main branch. 

Another alternative is to set this to run when a new tag is made.